tabula/io.py

```diff
@@ -13,8 +13,8 @@
 
 Example:
 
-    >>> import tabula
-    >>> df = tabula.read_pdf("/path/to/sample.pdf", pages="all")
+>>> import tabula
+>>> df = tabula.read_pdf("/path/to/sample.pdf", pages="all")
```

Comment:
Can you keep this indent as well?

```suggestion
    >>> import tabula
    >>> df = tabula.read_pdf("/path/to/sample.pdf", pages="all")
```

Reply:
Sure. I just updated the indents at the required lines.